dev/integration_tests/ios_platform_view_tests/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -564,14 +564,12 @@
 				CURRENT_PROJECT_VERSION = 1;
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
`PlatformViewUITests` target should not be overriding the project-level target version or device family.